static/js/settings_org.js

```diff
@@ -294,17 +294,19 @@ exports.populate_auth_methods = function (auth_methods) {
         }));
     });
     loading.destroy_indicator($('#admin_page_auth_methods_loading_indicator'));
-    if (!page_params.is_admin) {
-        var tip_box = $("<div class='tip'></div>")
-            .text(i18n.t("Only organization administrators can edit these settings."));
-        // Don't prepend a tip to custom emoji settings page. We handle it separately.
-        $(".organization-box").find(".settings-section:not(.can-edit)")
-            .not("#emoji-settings")
-            .not("#user-groups-admin")
-            .prepend(tip_box);
-    }
 };
 
+function insert_tip_box() {
+    if (page_params.is_admin) {
+        return;
+    }
+    var tip_box = templates.render("organization-settings-tip", {is_admin: page_params.is_admin});
+    $(".organization-box").find(".settings-section:not(.can-edit)")
+        .not("#emoji-settings")
+        .not("#user-groups-admin")
+        .prepend(tip_box);
+}
+
```

Comment:
This commit had 3 typos in its commit message -- please proofread your commit messages more carefully :)